vaccine_feed_ingest/runners/me/maine_gov/parser.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
```

Comment:
Please name this file `parse.py` so it gets picked up by the cli!

Reply:
Doh, muscle memory!  Fixed.